### DISCUSSION TITLE
Appium Server terminates during the test execution after average of 30 sec starting Appium Server

### DISCUSSION THREAD

**Discussion by chathunm** [appium/appium#19015] [Q&A]
Appium version 2.0.1
Desktop OS/version used to run Appium: Windows 10
Node version : 16.20.1
Real device or emulator/simulator: Simulator

I'm testing flutter application using appium flutter driver. Firstly I start Appium server and then execute my tests. Server started properly and during the test execution my app is loading as expected and doing some of the test actions in the test script. After a while (approx 30 sec) appium server terminates without any reason.

Test script is independent on this case as I'm just repeating some back and forth actions in the app.

May I know the reason for this? 
Appium Log File : 
[appium_log.txt](https://github.com/appium/appium/files/12391440/appium_log.txt)

**Comment by jlipps** [comment]
Please create a bug report instead.
